Jenkinsfile

```diff
@@ -1049,6 +1080,19 @@ pipeline {
                         buildHipClangJobAndReboot(setup_flags: Full_test, build_cmd: Navi21_build_cmd, build_install: "true")
                     }
                 }
+                stage('Fp32 Hip All Install navi32') {
+                    when {
+                        beforeAgent true
+                        expression { params.TARGET_NAVI32 && params.DATATYPE_FP32 }
+                    }
+                    options {
+                        retry(2)
+                    }
+                    agent{ label rocmnode("navi32") }
+                    steps{
+                        buildHipClangJobAndReboot(setup_flags: Full_test, build_cmd: Navi21_build_cmd, build_install: "true")
```

Comment:
[Q] I suspect that `Navi21_build_cmd` (which is a W/A for a Navi21-specific SWDEV-290754) is not necessary for both Navi 21 and 32. Or I am wrong?